specification/validated-relationships-service-api.yaml

```diff
@@ -149,6 +149,37 @@ servers:
   - url: 'https://api.service.nhs.uk/validated-relationships/FHIR/R4'
     description: Production environment.
 paths:
+  /QuestionnaireResponse:
```

Comment:
I presume this is ok being above `/RelatedPerson`. Should the list be in alphabetical order?

Reply:
I put it above RelatedPerson for alphabetical order. I don't think there is a strict rule.

Reply:
I should have remembered that R comes after Q 😞